spec/vector_spec.rb

```diff
@@ -1966,6 +1966,24 @@ def expect_correct_tokens hash
     end
   end
 
+  context "#sort_by_index" do
+    let(:asc) { vector.sort_by_index }
+    let(:desc) { vector.sort_by_index(ascending: false) }
+
+    context 'numeric vector' do
+      let(:vector) { Daru::Vector.new [11, 13, 12], index: [23, 21, 22] }
+      specify { expect(asc.to_a).to eq [13, 12, 11] }
```

Comment:
Why use this syntax and not the `its {expected.... }` type of syntax that is present in the other examples?